winsup/utils/getprocaddr.c

```diff
@@ -0,0 +1,198 @@
+#include <stdio.h>
+#include <windows.h>
+
+/**
+ * To determine the address of kernel32!CtrlRoutine, we need to use
+ * dbghelp.dll. But we want to avoid linking statically to that library because
+ * the normal operation of cygwin-console-helper.exe (i.e. to allocate a hidden
+ * Console) does not need it.
```

Comment:
This comment is now super stale. Likewise the next paragraph.

Reply:
Updated https://github.com/msys2/msys2-runtime/pull/31/commits/8e00f73dd4b43e218316faf604ffd001f3c7e4bf